### ISSUE TITLE
Compilation of old errors that need to be checked by people

### ISSUE THREAD

**Issue by MichaelDavidJohnson** [vn-ki/anime-downloader#462]
Hi guys, I just did a purge of all of the old inactive errors. The ones I did not immediately know a fix for I've compiled in the list below; 

- Option to resume failed downloads
- Adding some way to filter eg filler lists that @ArjixGamer mentioned in the discord
- disable ssl not carrying to aria
- downloading from watch re-downloads instead of skipping previously downloaded eps
- can't get weird episode names, like censored and uncensored 
- `anime dl https://kisscartoon.is/Cartoon/Mia-and-Me-Season-3 -e 15:26 -ll DEBUG -xd "{aria2}" ` fix this command
-  animepahe NoneType groups error is back
-  Handling special characters in anime names, K-ON! and K-ON!!
-  Adding tracking to watch 
-  Improve quality handling
-  KissAnime picks lowest quality?
-  KissAnime using RapidVideo when RapidVideo is broken, Unfortunately this must be delegated to Arjix and Red Guy as they re-implemented KissAnime. They'll know this answer

If anyone has any free time, consider working on these issues.

**Comment by nate-moo** [comment]
> anime dl https://kisscartoon.is/Cartoon/Mia-and-Me-Season-3 -e 15:26 -ll DEBUG -xd "{aria2}" fix this command

Isn’t -ll DEBUG in the wrong place? Shouldn’t it be before dl?

**Comment by MichaelDavidJohnson** [comment]
> > anime dl https://kisscartoon.is/Cartoon/Mia-and-Me-Season-3 -e 15:26 -ll DEBUG -xd "{aria2}" fix this command
> 
> Isn’t -ll DEBUG in the wrong place? Shouldn’t it be before dl?

Yes, put it in the correct place and it still errors. The line was from a really old error where -ll DEBUG could be called like that.

**Comment by ArjixGamer** [comment]
so back then the log level was handled by dl.py?

**Comment by vn-ki** [comment]
> so back then the log level was handled by dl.py?

There was no dl.py back then.

**Comment by Blatzar** [comment]
> KissAnime picks lowest quality?
Should be using hydrax which got fixed to use quality based on config
Either way doesnt really matter as kissanime implemented hcaptcha which pops up pretty often. Kissanime should be disabled or selenium adds support for hcaptcha (brings up non headless for users).

**Comment by hnumair** [comment]
>     * Option to resume failed downloads
If we can pass arguments for aria2c, we can pass --always-resume=true to always resume the download from .aria2 file.

**Comment by Blatzar** [comment]
:heavy_check_mark: **Option to resume failed downloads**
Pretty sure this is already the case

:heavy_check_mark: **Adding some way to filter eg filler lists that @ArjixGamer mentioned in the discord**
`--skip-fillers` got added 

:heavy_check_mark: **disable ssl not carrying to aria**
Pretty sure aria2 downloads regardless of ssl

**downloading from watch re-downloads instead of skipping previously downloaded eps**
I think this is due to different filenames, will investigate

**can't get weird episode names, like censored and uncensored**
I'll need some further explaining

**anime dl https://kisscartoon.is/Cartoon/Mia-and-Me-Season-3 -e 15:26 -ll DEBUG -xd "{aria2}" fix this command**
Will do

:heavy_check_mark:  **animepahe NoneType groups error is back**
Deactivated, but will be investigated once m3u8 downloader is back

**Handling special characters in anime names, K-ON! and K-ON!!**
Willfix

**Adding tracking to watch**
Leaving this to @IguanasInPyjamas 

**Improve quality handling**
Too vauge, it seems to work fine already

:heavy_check_mark:  **KissAnime picks lowest quality?
KissAnime using RapidVideo when RapidVideo is broken, Unfortunately this must be delegated to Arjix and Red Guy as they re-implemented KissAnime. They'll know this answer**
Dead
